### PR TITLE
Set deployment schedule for BIP 68,112,113

### PR DESCRIPTION
**Pull request by btcdrak**

### CODE REVIEW

File: bip-0068.mediawiki [hunk 1]
```diff
@@ -218,9 +218,13 @@ This BIP was edited by BtcDrak, Nicolas Dorier and kinoshitajona.
...
 
-This BIP is to be deployed by either version-bits BIP9 or by isSuperMajority(). Exact details TDB.
+This BIP is to be deployed by "versionbits" BIP9 using bit 0.
 
-It is recommended to deploy BIP112 and BIP113 at the same time.
+For Bitcoin '''mainnet''', the BIP9 '''starttime''' will be midnight 1st May 2016 UTC (Epoch timestamp 1462060800) and BIP9 '''timeout''' will be midnight 1st May 2017 UTC (Epoch timestamp 1493596800).
+
+For Bitcoin '''testnet''', the BIP9 '''starttime''' will be midnight 1st March 2016 UTC (Epoch timestamp 1456790400) and BIP9 '''timeout''' will be midnight 1st May 2017 UTC (Epoch timestamp 1493596800).
+
+This BIP must be deploy simultaneously with BIP112 and BIP113 using the same deployment mechanism.
```
Comment (morcos) [inline bip-0068.mediawiki]: "This BIP will be deployed simultaneously..."

**Comment by morcos** [comment]
ACK modulo nit on deployment wording in all 3 BIPs (just commented on the first one)

**Comment by btcdrak** [comment]
@morcos nit fixed.

**Comment by luke-jr** [comment]
Note that ACK/NACK here is a strict boolean, and relevant only from the BIP(s) Authors.

In this case, since btcdrak is listed for 112 and 113, this just needs an ACK from either @maaku or @afk11

**Comment by afk11** [comment]
ACK @ 63660c0

**Pull request merged**